vpp/vaapipostprocess_base.cpp

```diff
@@ -70,6 +65,28 @@ void VaapiPostProcessBase::cleanupVA()
     m_display.reset();
 }
 
+YamiStatus VaapiPostProcessBase::queryVideoProcFilterCaps(
+    VAProcFilterType filterType, void* filterCaps, uint32_t* numFilterCaps)
+{
+    if (!filterCaps)
```

Comment:
suppose have a case: the caller used 2 steps to get all Caps:
step1: Used  filterCaps == NULL && numFilterCaps != NULL to get accurate number of filter Caps.
       uint32_t numFilterCaps;
       queryVideoProcFilterCaps(filterType, NULL, &numFilterCaps);
step 2: 
      filterCaps = malloc(numFilterCaps * sizeof(xxx));
      queryVideoProcFilterCaps(filterType,  filterCaps,  NULL);

if not, the caller maybe always use this function like this:
    
 filterCaps [MAX_NUM_FILTER_CAPS];
 queryVideoProcFilterCaps(filterType,  filterCaps, &numFilterCaps);
 
 It is not a friendly interface to the caller if can't support the filterCaps == NULL && numFilterCaps != NULL case. :)